incubator-tools/amount_rectification_from_words_to_numbers/amount_rectification_from_words_to_numbers.ipynb

```diff
@@ -0,0 +1,709 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        # print("Detected source language: {}".format(result["detectedSourceLanguage"]))
</pre><p>There's a lot of commented out code in this notebook. Please remove</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/document-ai-samples/pull/720/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
I don't see an updated notebook. @bharadwajreddyvangimalla There aren't any additional commits after my comments.

Reply:
The incubator team has shared the edits, I will be working on the new commit and pushing it here.